orttraining/orttraining/python/training/_ortmodule_graph_execution_manager.py

```diff
@@ -89,6 +89,8 @@ def __init__(self, module):
         self._propagate_cast_ops_level = -1
         # List of opcodes to be considered safe to move before/after cast operation if propagate_cast_ops_level is zero.
         self._propagate_cast_ops_allow = []
+        # Whether allow fusion of layer norm subgraph if doing so will cause modified precision.
+        self._allow_layer_norm_mod_precision = False
```

Comment:
>_allow_layer_norm_mod_precision [](http://example.com/codeflow?start=13&length=31)

curious about any negative impact if default is true. Also this flag is applicable to fp16 only, right?

Reply:
The negative impact would be lower perf and higher activation memory usage as we would skip fusing the LN subgraph. But since fusing in the presence of this kind of a subgraph results in a different computation, we should not turn it on by default. And yes, this is applicable for a mixedprecision graph.